src/HostManager.cpp

```diff
@@ -60,28 +60,6 @@ bool HostManager::deleteHost( QString hostname )
     }
 }
 
-// Not Used - if resurrected MUST re-write to use mPoolReadWriteLock.lockForWrite() as in deleteHost(...)/addHost(...)!
```

Comment:
I commented out these methods because they were currently unused but would need reworking to be usable.  That does not necessarily mean that they will not be wanted in the future so I did not remove them.  Unless @vadi2 wants them culled I would prefer to leave them in place.